src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json

```diff
@@ -20,17 +20,18 @@
   },
 
   "frameworks": {
-    "dnxcore50": {
-      "imports": "portable-net45+win8",
+    "netstandard1.5": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45+win8"
+      ],
       "dependencies": {
-        "NETStandard.Library": "1.0.0-rc3-23904",
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc2-*"
```

Comment:
Although I gave you this version string for testing, I recommend you use "1.0.0-rc2-3*" for the string here as it will ensure you are only getting the newest versions of this package (the older ones will break)